standard/info/wikis/geoguessr/info.lua

```diff
@@ -0,0 +1,42 @@
+---
+-- @Liquipedia
+-- wiki=geoguessr
+-- page=Module:Info
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+return {
+	startYear = 2013,
+	wikiName = 'geoguessr',
+	name = 'GeoGuessr',
+	defaultGame = 'geoguessr',
+	games = {
+		geoguessr = {
+			abbreviation = 'Geo',
+			name = 'GeoGuessr',
+			link = 'GeoGuessr',
+			logo = {
+				darkMode = 'GeoGuessr allmode.png',
+				lightMode = 'GeoGuessr allmode.png',
+			},
+			defaultTeamLogo = {
+				darkMode = 'GeoGuessr allmode.png',
+				lightMode = 'GeoGuessr allmode.png',
+			},
+		},
+	},
+	config = {
+		squads = {
+			hasPosition = false,
+			hasSpecialTeam = false,
+			allowManual = false,
+		},
+		match2 = {
+			matchWidthMobile = 110,
+			matchWidth = 190,
```

Comment:
Are these needed?

Reply:
can kick these two custom width, player names are not too long to need extended width (i was just copy the values to fix the error)

Reply:
Then you can remove them